stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/usecase/QueryMembersImpl.kt

```diff
@@ -0,0 +1,50 @@
+package io.getstream.chat.android.livedata.usecase
+
+import androidx.annotation.CheckResult
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.call.Call
+import io.getstream.chat.android.client.call.CoroutineCall
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.android.client.utils.FilterObject
+import io.getstream.chat.android.client.utils.Result
+import io.getstream.chat.android.livedata.ChatDomainImpl
+
+public interface QueryMembers {
+
+    @CheckResult
+    public operator fun invoke(
+        channelType: String,
+        channelId: String,
```

Comment:
I think we can replace `channelType` and `channelId` with `cid` to match other use cases